modules/proc_upgrade_standalone.adoc

```diff
@@ -64,15 +64,88 @@ Images for Quay 3.3.4 and earlier are available from link:https://quay.io[quay.i
 
 == Upgrade to 3.9.z from 3.8.z
 
-If you are upgrading your standalone {productname} deployment from 3.8.z -> 3.9, it is highly recommended that you upgrade PostgreSQL from version 10 -> 13. To upgrade PostgreSQL from 10 -> 13, you must bring down your PostgreSQL 10 database and run a migration script to initiate the process:
+If you are upgrading your standalone {productname} deployment from 3.8.z -> 3.9, it is highly recommended that you upgrade PostgreSQL from version 10 -> 13. To upgrade PostgreSQL from 10 -> 13, you must bring down your PostgreSQL 10 database and run a migration script to initiate the process. 
+
+Use the following procedure to upgrade PostgreSQL from 10 -> 13 on a standalone {productname} deployment. 
+
+.Procedure 
+
+. Enter the following command to scale down the {productname} container:
++
 [source,terminal]
 ----
-$ podman run -d --name postgresql_database \
+$ sudo podman stop <quay_container_name>
+----
+
+. Optional. If you are using Clair, enter the following command to stop the Clair container:
++
+[source,terminal]
+----
+$ sudo podman stop <clair_container_id>
+----
+
+. Run the Podman process from SCLOrg's link:https://github.com/sclorg/postgresql-container/tree/master/13#data-migration[Data Migration] procedure, which allows for data migration from a remote PostgreSQL server:
++
+[source,terminal]
+----
+$ sudo podman run -d --name <current_postgresql_database> <1>
     -e POSTGRESQL_MIGRATION_REMOTE_HOST=172.17.0.2 \
     -e POSTGRESQL_MIGRATION_ADMIN_PASSWORD=remoteAdminP@ssword \
+    -v </host/data/directory:/var/lib/postgresql/data> <2>
     [ OPTIONAL_CONFIGURATION_VARIABLES ]
     rhel8/postgresql-13
 ----
++
+<1> The name of your PostgreSQL 10 database.
+<2> You must specify a different volume mount point than the one from your initial PostgreSQL 10 deployment. This prevents data from being overwritten by the new container. 
+
+. Optional. If you are using Clair, repeat the previous step for the Clair PostgreSQL database container. 
+
+. Run the PostgreSQL 13 container, using the mount points written from Step 3:
++
+[source,terminal]
+----
+$ sudo podman run -d --rm --name postgresql-quay \
+  -e POSTGRESQL_USER=<username> \
+  -e POSTGRESQL_PASSWORD=<password> \
+ 	 -e POSTGRESQL_DATABASE=<quay_database_name> \
+  -e POSTGRESQL_ADMIN_PASSWORD=<admin_password> \
+  -p 5432:5432 \
+  -v </host/data/directory:/var/lib/postgresql/data> \
+    registry.redhat.io/rhel8/postgresql-13:1-109
+----
+
+. Optional. If you are using Clair, repeat the previous step for the Clair PostgreSQL database container.
+
+. Start the {productname} container:
++
+[source,terminal]
+----
+$ sudo podman run -d --rm -p 80:8080 -p 443:8443 --name=quay \
+-v /home/stevsmit/quay-poc/config:/conf/stack:Z \
+-v /home/stevsmit/quay-poc/storage:/datastorage:Z \
+{productrepo}/{quayimage}:{productminv}0
+----
+
+. Optional. Restart Clair by using the container image, using the mount points you used in Step 4:
```

Comment:
If also run the postgres migration for Clair DB, the new data volume is mount to Clair Postgres 13, hence the Clair config is not changed, no need to mount the new volume to Clair Container.